pages/agent/v3/queues.md

```diff
@@ -23,6 +23,9 @@ buildkite-agent start --tags "queue=building,queue=testing"
 
 <%= image "agent-queues.png", width: 1182/2, height: 160/2, alt: "Screenshot of an agent's tags showing both building and testing queues" %>
 
+>🚧 Configuring Cluster Queues
+>If you have [Clusters](/docs/agent/clusters) enabled, and are configuring your agent with a _Cluster Queue_, you will need to ensure the cluster queue has been explicitly created first. See [setting up Cluster Queues](/docs/agent/clusters#set-up-a-cluster-set-up-queues) to learn more.
```

Comment:
```suggestion
>If you have [Clusters](/docs/agent/clusters) enabled and are configuring your agent with a _Cluster Queue_, you need to create the cluster queue first. See [Set up a Cluster Queue](/docs/agent/clusters#set-up-a-cluster-set-up-queues) for more information.
```

This is a very helpful note!

Couple little suggestions here:

- Shortened the sentence by making it more direct
- Tweaked the final sentence to match what appears to be our common pattern in these docs: "see <blah> for more information" (50 instances), or "for more details" (29 instances).

Reply:
@timriley great suggestions for making this more concise. I'm getting flashbacks to my uni days haha, I had one editor who always told me my sentences were too long.